recipe/emailpassword/recipeImplementation.go

```diff
@@ -54,7 +54,7 @@ func MakeRecipeImplementation(querier supertokens.Querier) epmodels.RecipeInterf
 		}
 		status, ok := response["status"]
 		if ok && status.(string) == "OK" {
-			user, err := parseUser(response["user"])
+			user, err := ParseUser(response["user"])
```

Comment:
You should not expose an internal helper function like this.. If this is required in testing, copy the implementation of that in the test folder.